app/controllers/chat_rooms_controller.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+# Chat_room
+class ChatRoomsController < ApplicationController
+  before_action :logged_in_user, only: %i[index create show destroy]
+  before_action :find_chat_room, only: %i[destroy add_confirm]
+
+  def index
+    show_users_and_rooms
+  end
+
+  def create
+    @chat_room = ChatRoom.create(title: params['chat_room']['title'])
+    @remember = Remember.create(user_id: current_user.id, chat_room_id: @chat_room.id)
+    respond_to(&:js)
+  rescue ActiveRecord::RecordNotDestroyed
```

Comment:
This Error only raised when t record failed the `destroy` method, Why use it to check `create` ?

Reply:
Fixed